File: Python/Uri1036_FormulaDeBhaskara.py
valores = input().split()

#a, b, c = map(float, input().split())
a = float(valores[0])
b = float(valores[1])
c = float(valores[2])

delta = ((b**2)-4*a*c)

if (delta < 0) or a == 0:
	print( "Impossivel calcular")
	
elif delta == 0:
	r1 = (-b + delta **(1/2))/(2*a)
	r2 = r1
	print("R1 = %.5f" %(r1))
	print("R2 = %.5f" %(r2))

else:
	r1 = (-b + delta **(1/2))/(2*a)
	r2 = (-b - delta **(1/2))/(2*a)
	print("R1 = %.5f" %(r1))
	print("R2 = %.5f" %(r2))

File: Javascript/uri1036_FormulaDeBhaskara.js
var input = require("fs").readFileSync("stdin", "utf8");
var lines = input.split(" ");

var a = parseFloat(lines.shift());
var b = parseFloat(lines.shift());
var c = parseFloat(lines.shift());
var t = parseFloat(lines.shift());

 if(((b * b) - 4 * a * c) < 0 || a == 0){
     console.log("Impossivel calcular");
 }
 else{
    t = Math.sqrt((b * b) - 4 * a * c);
    console.log("R1 = " + ((-b + t) / (2 * a)).toFixed(5));
    console.log("R2 = " + ((-b - t) / (2 * a)).toFixed(5));
 }
